tests/_benderjs/ckeditor/static/tools.js

```diff
@@ -1080,7 +1081,7 @@
 					'setUp': 1,
 					'tearDown': 1
 				},
-				i, editorName;
+				i, editorName, postfix = 1;
```

Comment:
The `postfix` could be something like `editorNumber`/`editorCounter` which is more meaningful.